manifests/t/Tencent/Effidit/0.3.0/Tencent.Effidit.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.7-4-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Tencent.Effidit
+PackageVersion: 0.3.0
+InstallerType: msi
+Scope: machine
+UpgradeBehavior: install
+FileExtensions:
+- ini
+- log
+- md
+- txt
+AppsAndFeaturesEntries:
+- ProductCode: '{428A0F86-31C0-4BDF-8E44-15FB5B32D807}'
+  UpgradeCode: '{71C40863-F454-4671-817C-15BE8C263DFC}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://effidit.qq.com/effidit-v0.3.0-zh-setup.msi
+  InstallerSha256: 08E12A9945C115DD77FCC8AB4F497464E84AD20C7FD4775BB5FC966928764B27
+  ProductCode: '{428A0F86-31C0-4BDF-8E44-15FB5B32D807}'
```

Comment:
Should you add the InstallerLocale and also `https://effidit.qq.com/effidit-v0.3.0-en-setup.msi` ?